tools/pipelines/templates/1ES/build-docker-service.yml

```diff
@@ -415,7 +415,7 @@ extends:
           - task: 1ES.PushContainerImage@1
             displayName: Docker Push
             inputs:
-              image: ${{ variables.buildContainerName }}:$(containerTagSuffix)
+              image: $(containerRegistryUrl)/${{ variables.buildContainerName }}:$(containerTagSuffix)
```

Comment:
I think `buildContainerName` already has a "prefix" that acts as registry. See how we define it [here](https://github.com/microsoft/FluidFramework/blob/a3dfbb6ac3c7c8893734907eaeef13d03b149d2b/tools/pipelines/templates/build-docker-service.yml#L108-L113).

Reply:
Updated.